docker/docker_image_src.go

```diff
@@ -161,6 +164,26 @@ func newImageSourceAttempt(ctx context.Context, sys *types.SystemContext, logica
 		client.Close()
 		return nil, err
 	}
+
+	if h, err := sysregistriesv2.AdditionalLayerStoreAuthHelper(endpointSys); err == nil && h != "" {
+		acf := map[string]struct {
+			Username      string `json:"username,omitempty"`
+			Password      string `json:"password,omitempty"`
+			IdentityToken string `json:"identitytoken,omitempty"`
```

Comment:
```suggestion
			IdentityToken string `json:"identityToken,omitempty"`
```
wouldn’t hurt

Reply:
This follows the coding style of other part of the codebase (pkg/docker/config/config.go uses `identitytoken`) : https://github.com/containers/image/blob/984c624c4c9e07c79bcdbc49679ba2a781805303/pkg/docker/config/config.go#L30

Reply:
The data structure in `pkg/docker/config` matches `github.com/docker/cli/config/types.AuthConfig`, to be interoperable with `docker login` and the like.

This is a distinct data structure (e.g. it doesn’t hide the user name / password in a base64-encoded string — and that’s good!), so we have the liberty, and the ~unique opportunity, to make it more convenient.

Reply:
Fixed to make it `identityToken`.